client/build.gradle

```diff
@@ -0,0 +1,21 @@
+dependencies {
+    implementation project(":common")
+    implementation 'org.springframework.boot:spring-boot-starter'
+    implementation 'org.springframework.cloud:spring-cloud-starter-openfeign'
```

Comment:
I think only openfeign itself is used in current code, the "spring cloud"-related feature (e.g. @FeignClient) are not used at all. Is that intended?

Reply:
about the @FeignClient, I don't want to make the UT to be a SpringBootTest, so I use the `OrderCommandClientConfig` to provide feignClientImpl instead of auto-config. Then I can pass the `OrderCommandClientConfig` into `@ContextConfiguration` in the UT.

but I am not sure what else features are provided by `spring-cloud-starter-openfeign` (maybe we can use in the future?), so I still use this dependency instead of the `openfeign`

Reply:
As I see it, the main purpose of spring cloud openfeign is using a controller interface as a feign client. It supports spring MVC RequestMapping which is not handled by feign itself.